components/LocationsList/index.js

```diff
@@ -1,34 +1,27 @@
 import styled from "styled-components";
 import LocationPreviewCard from "@/components/LocationPreviewCard";
+import Heading from "@/components/Heading";
+import TitleSection from "@/components/TitleSection";
+import NavBar from "../NavigationsBar";
 
 const StyledUl = styled.ul`
-  padding: 1px;
-  margin: 5px;
+  padding: 0;
+  margin-top: 80px;
```

Comment:
I think you need a ``margin-bottom: 80px;`` as well, because the last entry in the list is not shown completely.

Reply:
Thank`s! Done!